api/ratelimit.proto

```diff
@@ -0,0 +1,102 @@
+syntax = "proto3";
+
+option go_package = "ratelimit";
+
+option cc_generic_services = true;
+
+package envoy.api.v1;
```

Comment:
We can chat more in a bit, but I think it would make sense to just put in the v2 namespace, rename to the new names. The old v1 API an continue to live in the existing location.